### PR TITLE
4 contact page

### PR DESCRIPTION
**Pull request by fabiopacifici**
Added route and view for the contact page

created a contacts.blade.php and added route inside web.php

### CODE REVIEW

File: resources/views/contacts.blade.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@extends('layouts.app')
+
+@section('content')
+<h1>Contacts page</h1>
```
Comment (fbhood) [inline resources/views/contacts.blade.php]: Can we change the h1 content to just Contact us?

File: routes/web.php [hunk 2]
```diff
@@ -16,3 +16,6 @@
 Route::get('/', function () {
     return view('welcome');
 });
+Route::get('/contacts', function () {
+    return view('contacts');
+});
```
Comment (fbhood) [inline routes/web.php]: add a name to the route

**Review by fbhood — CHANGES_REQUESTED** [review]
Please make requested changes

File: resources/views/contacts.blade.php (hunk 1 above)
Comment (fabiopacifici) [reply resources/views/contacts.blade.php]: ok working on it!

File: routes/web.php (hunk 2 above)
Comment (fabiopacifici) [reply routes/web.php]: ok will do!

**Review by fbhood — APPROVED** [review]
ok thanks for your work

**Pull request merged**